<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetLogistica</class>
 <widget class="QWidget" name="WidgetLogistica">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="frameForn">
      <layout class="QGridLayout" name="gridLayout_34">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="labelCompras_11">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Fornecedores</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="TableView" name="tableForn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QTabWidget" name="tabWidgetLogistica">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Coleta</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="WidgetLogisticaColeta" name="widgetColeta" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Recebimento</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_28">
        <item row="0" column="0">
         <widget class="WidgetLogisticaRecebimento" name="widgetRecebimento" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Entregas</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_27">
        <item row="0" column="0">
         <widget class="WidgetLogisticaEntrega" name="widgetEntrega" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>src/tableview.h</header>
  </customwidget>
  <customwidget>
   <class>WidgetLogisticaColeta</class>
   <extends>QWidget</extends>
   <header>src/widgetlogisticacoleta.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetLogisticaRecebimento</class>
   <extends>QWidget</extends>
   <header>src/widgetlogisticarecebimento.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetLogisticaEntrega</class>
   <extends>QWidget</extends>
   <header>src/widgetlogisticaentrega.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
